operator/pkg/helmreconciler/apply.go

```diff
@@ -193,8 +193,7 @@ func (h *HelmReconciler) ApplyObject(obj *unstructured.Unstructured, serverSideA
 	}
 
 	gvk := obj.GetObjectKind().GroupVersionKind()
-
-	if serverSideApply {
+	if serverSideApply && obj.GetKind() != name.CRDStr {
```

Comment:
Can you add a comment with TODO linking to issue?

Reply:
added